recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
 MACOSX_SDK_VERSION:            # [osx and x86_64]
-  - 10.15                      # [osx and x86_64]
+  - 11.0                       # [osx and x86_64]
```

Comment:
Please read [this]( https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks )